secp256k1-sys/src/lib.rs

```diff
@@ -517,11 +526,32 @@ impl core::hash::Hash for Keypair {
     }
 }
 
+/// Library-internal representation of a ElligatorSwift encoded group element.
+#[repr(C)]
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct ElligatorSwift([u8; 64]);
+
+impl ElligatorSwift {
+    pub fn from_array(arr: [u8; 64]) -> Self {
```

Comment:
Since this is public maybe we should document that the input array should typically be the array that was created using `secp256k1_ellswift_create`. That is the case, right?

Reply:
In theory, it could be any 64-bytes array. Ofc we assume that someone knows the secret key associated with the encode point, but this array could come over wire, for example. Not sure if it is restricted to things we explicitly created with `secp256k1_ellswift_create`.